src/linker/Linker/UnconditionalSuppressMessageAttributeState.cs

```diff
@@ -173,23 +174,26 @@ public void LookForModuleLevelAndGlobalSuppressions (ModuleDefinition module, IC
 				if (!TryDecodeSuppressMessageAttributeData (instance, out info))
 					continue;
 
-				if (info.Target == null && (info.Scope == "module" || info.Scope == null)) {
+				// If the scope is missing we treat the suppression as if it was placed on the module.
+				var scope = info.Scope?.ToLower ();
+				if (info.Target == null && (scope == "module" || scope == null)) {
 					AddSuppression (info, provider);
 					continue;
 				}
 
-				if (info.Target == null || info.Scope == null)
+				if (info.Target == null || scope == null)
 					continue;
```

Comment:
Should null Target or Scope be warnings too?

Reply:
I don't see why we should. Both parameters are only relevant for global suppressions, which should always look like `[assembly: Unc...`/`[module: Unc...`, and then I think it's reasonable to put the suppression on the module whenever neither are specified. Having a non-module scope without a target could be a case where I think a warning makes sense, e.g. `[module: USM("", "", Scope = "type")]`, I haven't tried this but I think this would actually throw from DocumentationSignatureParser :/

Reply:
We should add tests for all these cases - to validate the current behavior. And if some of those behaviors seem broken, either fix it or file new bugs. But have them "verified" by tests regardless.

Reply:
Added some tests and changed the DocumentationSignatureParser so that we don't throw when `module` or `id` are null.